docs/en_us/enrollment_api/Makefile

```diff
@@ -0,0 +1,164 @@
+# Makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    ?=
+SPHINXBUILD   ?= sphinx-build
+PAPER         ?=
+BUILDDIR      ?= build
+
+# User-friendly check for sphinx-build
+ifeq ($(shell which $(SPHINXBUILD) >/dev/null 2>&1; echo $$?), 1)
+$(error The '$(SPHINXBUILD)' command was not found. Make sure you have Sphinx installed, then set the SPHINXBUILD environment variable to point to the full path of the '$(SPHINXBUILD)' executable. Alternatively you can add the directory with the executable to your PATH. If you don't have Sphinx installed, grab it from http://sphinx-doc.org/)
```

Comment:
Suggestion: Alternatively, you can add the directory that contains the executable to your PATH. If you don't have Sphinx installed, you can obtain it from http://sphinx-doc.org.

Not sure what PATH refers to - the path of the executable, as mentioned in the previous sentence? Can we add a clarifying noun after PATH?